charts/chainlink-cluster/devspace.yaml

```diff
@@ -2,75 +2,21 @@ version: v2beta1
 name: chainlink
 
 vars:
-  NS_TTL: 72h
-  DEVSPACE_IMAGE:
-    noCache: true
-    source: env
-  # This is the base domain in AWS Route 53 that our ingress subdomains will use.
-  DEVSPACE_INGRESS_BASE_DOMAIN:
-    source: env
-  # This is the ARN of the AWS ACM certificate that will be used for the ingress.
-  DEVSPACE_INGRESS_CERT_ARN:
-    source: env
-  # This is a comma separated list of CIDR blocks that will be allowed to access the ingress.
-  DEVSPACE_INGRESS_CIDRS:
-    source: env
-  # Time to wait for pods to be in `Ready` condition
-  DEVSPACE_K8S_POD_WAIT_TIMEOUT: 600s
+  DEVSPACE_ENV_FILE: .env
 
 # This is a list of `pipelines` that DevSpace can execute (you can define your own)
 pipelines:
-  dev:
```

Comment:
we don't need this anymore, because build process is faster now, right?

Reply:
I'd say let's remove it for the sake of simplicity

Reply:
both, @skudasov tested and concluded that "hot reloading" is actually slower than pod replacement due to bandwidth requirements